extensions/amp-flowplayer/0.1/test/validator-amp-flowplayer.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
```

Comment:
Please add the copyright statement to the top of this test file. You can look at any other validator test for a similar statement.

Reply:
Copyright statement added

Reply:
You will need to rerun the validator update_tests command to pick the same up in the corresponding `.out` file.

Reply:
I also run the validator to update .out file.

Reply:
@Gregable please re-review